jupyter_console/completer.py

```diff
@@ -7,6 +7,16 @@
 from traitlets.config import Configurable
 from traitlets import Float
 
+import jupyter_client
+
+
+# jupyter_client 7.0+ has async channel methods that we expect to be sync here
+if jupyter_client._version.version_info[0] >= 7:
```

Comment:
You can also just do 

```suggestion
if jupyter_client.version_info >= (7,):
```

Since Python 3 you can compare tuples of different length, and it avoid accessing  aprivate field

Reply:
Thanks!